caleb-lab/server.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const express = require('express');
+const morgan = require('morgan');
+const jsonParser = require('body-parser').json();
+
+const PORT = process.env.PORT || 3000;
+const app = express();
+const router = express.Router();
+const cors = require('./lib/cors');
+
+app.use(morgan('dev'));
+app.use(jsonParser);
+app.use(cors);
+//there are different plugins for morgan that will give us differently formatted output
```

Comment:
**TODO FOR LOST POINTS**

Take out your comments or put them in another branch for you own reference.